### ISSUE TITLE
Update or replace jasmine-core

### ISSUE THREAD

**Issue by Jawayria** [edx/upgrades#47]
We use the package [jasmine-core](https://github.com/jasmine/jasmine) in edx/edx-enterprise. It is a testing framework for Javascript and it hasn't yet added support for Django3.2. Please follow the guidance in https://openedx.atlassian.net/wiki/spaces/AC/pages/3036972032/Handling+Outdated+Dependencies to resolve the problem this poses for the Open edX Django 3.2 upgrade.

**Comment by Squirrel18** [comment]
Hi, @natabene I would like to contribute to this problem, could you please assign it to me?

**Comment by natabene** [comment]
@Squirrel18 Absolutely, thank you for reaching out to us and picking this up. I will assign this to you. Please keep us updated on your progress here.

**Comment by natabene** [comment]
@Squirrel18 Just checking if you had a chance to look into this.

**Comment by Squirrel18** [comment]
Hi @natabene, I don't know what happened but my feedback never got submitted, my fault. Yes, I did work on this and here is what I found:

Jasmine Core for Python is deprecated (https://jasmine.github.io/setup/python.html) so the options are:
1. Remove the Jasmine Core support and change it to the supported Jasmine Node package.
2. Remove the Jasmine Core support and run the JS test suites with Karma.

Either of those options is fine for running the JS test suites, let me know if you want to discuss or research about it. Again sorry for the late response.

**Comment by natabene** [comment]
@Squirrel18  Thank you for your research. @Jawayria @jmbowman Please advise on how to proceed.
